### DISCUSSION TITLE
Trying to cancel pro in favour of GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by julietrb1** [community/community#21658] [New to GitHub]
<p>Hi, great community. I’ve only just noticed that my GitHub Student Developer Pack perk expired at some point, and manually switched over to a paid pro subscription. I’ve now managed to renew the student pack, but it still says that I’m paying for pro.</p>
<p>If I downgrade to the free tier, I fear it’ll completely bar me from being able to take advantage of the student pack. Is there anything to be done here, or is it perhaps a waiting game?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>This is a situation our billing system doesn’t handle very well. Please <a href="https://support.github.com/contact">open a ticket</a> and the billing team will fix that properly for you.</p>

**Comment by such3** [comment]
> Hi, great community. I’ve only just noticed that my GitHub Student Developer Pack perk expired at some point, and manually switched over to a paid pro subscription. I’ve now managed to renew the student pack, but it still says that I’m paying for pro.
> 
> If I downgrade to the free tier, I fear it’ll completely bar me from being able to take advantage of the student pack. Is there anything to be done here, or is it perhaps a waiting game?

What happens after that? My plan expires, and I don't know if I need to downgrade or if it will automatically switch to a paid plan?
